Index: lib/main.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/main.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/main.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -17,7 +17,7 @@
 import 'provider/favs_provider.dart';
 import 'screens/auth/login.dart';
 import 'screens/auth/sign_up.dart';
-import 'screens/bottom_bar.dart';
+import 'screens/top_app_bar.dart';
 import 'screens/cart/cart.dart';
 import 'screens/feeds.dart';
 import 'screens/user_state.dart';
@@ -113,7 +113,7 @@
                         CategoriesFeedsScreen(),
                     LoginScreen.routeName: (ctx) => LoginScreen(),
                     SignUpScreen.routeName: (ctx) => SignUpScreen(),
-                    BottomBarScreen.routeName: (ctx) => BottomBarScreen(),
+                    TopBarScreen.routeName: (ctx) => TopBarScreen(),
                     UploadProductForm.routeName: (ctx) => UploadProductForm(),
                     ForgetPassword.routeName: (ctx) => ForgetPassword(),
                     OrderScreen.routeName: (ctx) => OrderScreen(),
Index: lib/screens/home.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/screens/home.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/screens/home.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -89,7 +89,8 @@
               height: 190.0,
               width: double.infinity,
               child: Carousel(
-                boxFit: BoxFit.fill,
+                boxFit: BoxFit.cover,
+                //boxFit: BoxFit.fill,
                 autoplay: true,
                 animationCurve: Curves.fastOutSlowIn,
                 animationDuration: Duration(milliseconds: 1000),
Index: lib/screens/landing_page.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/screens/landing_page.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/screens/landing_page.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -1,7 +1,7 @@
 import 'package:la_boutique_de_a_y_s_app/consts/colors.dart';
 import 'package:la_boutique_de_a_y_s_app/screens/auth/login.dart';
 import 'package:la_boutique_de_a_y_s_app/screens/auth/sign_up.dart';
-import 'package:la_boutique_de_a_y_s_app/screens/bottom_bar.dart';
+import 'package:la_boutique_de_a_y_s_app/screens/top_app_bar.dart';
 import 'package:la_boutique_de_a_y_s_app/services/global_method.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
@@ -124,8 +124,9 @@
         //   'https://image.flaticon.com/icons/png/128/564/564619.png',
         //   fit: BoxFit.contain,
         // ),
-        errorWidget: (context, url, error) => Icon(Icons.error),
+        errorWidget: (context, url, error) => Container(),
         fit: BoxFit.cover,
+        color: Colors.white,
         height: double.infinity,
         width: double.infinity,
         alignment: FractionalOffset(_animation.value, 0),
@@ -302,13 +303,13 @@
   }
 
   static FadeInImage buildLogo() {
-    var size = 500.0;
+    var size = 400.0;
     return FadeInImage(
       width: size,
       height: size,
       image: AssetImage('assets/logo/la-boutique-logo.gif'),
       placeholder: AssetImage('assets/logo/la-boutique-logo.gif'),
-      fit: BoxFit.cover,
+      fit: BoxFit.fill,
     );
   }
 }
Index: lib/screens/main_screen.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/screens/main_screen.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/screens/main_screen.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -1,13 +1,13 @@
 import 'package:la_boutique_de_a_y_s_app/screens/upload_product_form.dart';
 import 'package:flutter/material.dart';
 
-import 'bottom_bar.dart';
+import 'top_app_bar.dart';
 
 class MainScreens extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return PageView(
-      children: [BottomBarScreen(), UploadProductForm()],
+      children: [TopBarScreen(), UploadProductForm()],
     );
   }
 }
Index: lib/screens/bottom_bar.dart
===================================================================
--- lib/screens/bottom_bar.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/screens/top_app_bar.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -7,14 +7,14 @@
 import 'feeds.dart';
 import 'home.dart';
 
-class BottomBarScreen extends StatefulWidget {
+class TopBarScreen extends StatefulWidget {
   static const routeName = '/BottomBarScreen';
 
   @override
-  _BottomBarScreenState createState() => _BottomBarScreenState();
+  _TopBarScreenState createState() => _TopBarScreenState();
 }
 
-class _BottomBarScreenState extends State<BottomBarScreen>
+class _TopBarScreenState extends State<TopBarScreen>
     with SingleTickerProviderStateMixin {
   int _selectedPageIndex = 1;
   List<Object> pages;
@@ -65,7 +65,9 @@
   Widget build(BuildContext context) {
     return Scaffold(
       body: getTabBarView(_tabController),
+      backgroundColor: Colors.white,
       appBar: AppBar(
+        backgroundColor: Colors.white,
         bottom: TabBar(
           controller: _tabController,
           isScrollable: true,
Index: lib/widget/backlayer.dart
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/widget/backlayer.dart	(revision c2662d2b3af7bb5934b62db4374b0a6e906c98e3)
+++ lib/widget/backlayer.dart	(revision 7ea132582baae325e5f8b18f3d6de1aaac558eec)
@@ -111,19 +111,19 @@
                     ),
                   ),
                 ),
-                const SizedBox(height: 10.0),
+                const SizedBox(height: 8.0),
                 content(context, () {
                   navigateTo(context, Marketplace.routeName);
                 }, 'Marketplace', 0),
-                const SizedBox(height: 10.0),
+                const SizedBox(height: 8.0),
                 content(context, () {
                   navigateTo(context, CartScreen.routeName);
                 }, 'Carrito', 1),
-                const SizedBox(height: 10.0),
+                const SizedBox(height: 8.0),
                 content(context, () {
                   navigateTo(context, Marketplace.routeName);
                 }, 'Favoritos', 2),
-                const SizedBox(height: 10.0),
+                const SizedBox(height: 8.0),
                 content(context, () {
                   navigateTo(context, UploadProductForm.routeName);
                 }, 'Subir nuevo producto', 3),
